hubspot/connection/__init__.py

```diff
@@ -204,6 +204,7 @@ def _require_successful_response(response):
             raise exception_class(
                 error_data['message'],
                 error_data['requestId'],
+                error_data,
```

Comment:
I've added failureMessages to the schema in 672019f014c1c36c6a1327a120d9d36dda3376bb